### ISSUE TITLE
Bug Hunt

### ISSUE THREAD

**Issue by jonathanwalkerscripts** [jonathanwalkerscripts/OppChasers#119]
Create an acceptance test-style checklist with the following: 

(Title) Bug Hunt

**On websites/applications of your choosing, see if you can find bugs in the following categories:**

- [ ] Image not appearing correctly
- [ ] Link broken/goes to wrong page
- [x] Color Contrast not ADA Compliant
- [x] Missing Alt Text 
- [x] Present Alt Text (Show me a screenshot from Inspect!)
- [x] Any Bug with Data Fields
- [x] Design Flaw or Typo
- [x] Crash or Error Message
- [x] Google Lighthouse ADA Score of exactly 80
- [x] Any bug on CEO Works or a company you've worked with

Bonus: Use the inspect tool to change the headline of a major news story. Post a screenshot in the comments section.

**Comment by jonathanwalkerscripts** [comment]
Image not appearing correctly 

Link 

Screenshot
